src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RADIUS_DEFAULT = 10;
+    private static final String COLOR_DEFAULT = "WHITE";
+    protected Random random = new Random();
+
+    protected Figure getRandomFigure(int randomFigure) {
```

Comment:
```suggestion
    protected Figure getRandomFigure() {
        int randomFigure = random.nextInt(FIGURE_NUMBER);
```
you should randomize a number here
this method should not depend on parameters from the outside, after each invocation of this method the figure should be absolutely random